### DISCUSSION TITLE
[Feature Request] navigate through (sub)folders

### DISCUSSION THREAD

**Discussion by RalfKo** [PaulWoitaschek/Voice#1417] [Ideas]
in #809 you talk ab out the folder structure when adding multiple books.

Please also introduce a navigation through folders on the homescreen of the app: so that the user starts at the topmost folder, with each subfolder being represented by a "cover"-like tile (take the image from an existing "Folder.jpg" or make a collage of the first 4 embed cover artworks you may find in the nested files, or just write the name of the folder in the middle of an grey tile)

**Comment by marksz** [comment]
This would be helpful esp. since books organized by author only show the first book in the menu.  You have to look at the track names to see the later books, and all books show the image of the first one.

**Comment by thw0rted** [comment]
@marksz you aren't actually "organizing by author", you're creating one massive book, as far as Voice is concerned.  That's why I've been active on the linked issue trying to advocate for other ways of organizing.

**Comment by marksz** [comment]
Yes, well all the other audiobook players on my phone show all the books in subfolders of an author folder as separate books.  I've installed a bunch because it is easier if I'm listening to 3 or 4 books at once, to get back to where I left off if I use a different player for each book.  This is the way most people have been organizing mp3s for music by artist, then album, for a long time.  I've just stopped using voice for books where I have several by the same author.

**Comment by thw0rted** [comment]
You can go put a "thumbs up" on #809 or comment there, you're not the only one who doesn't like the "my way or the highway" approach to folder layout.

**Comment by marksz** [comment]
Done.  Thanks for pointing that out.

**Reply by gitoss** [reply]
This is my #1 issue with Voice, and the reason I refrain from using it again and again.

The flat or grid mode with a lot of audio books is not structured enough for a lot of audio books (like top folders / top categories), and without perfect tagging not useful to identify individual audio books.

I tope a "tree layout" in addition to the list and grid layout will be added sometime, even the date of this 2019 ticket doesn't seem promising.

Feature request: Add a "tree layout" in addition to the list layout and grid layout.***
